releasenotes/notes/handle-empty-hostname-0d530fcc86cba056.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    All python and go checks support the new ``empty_default_hostname`` option
```

Comment:
What about the `min_collection_interval`? It wasn't considered before?

Reply:
All python checks supported `min_collection_interval` since agent5, golang checks not supporting them was an oversight